user_tools/src/spark_rapids_pytools/resources/dataproc-configs.json

```diff
@@ -258,6 +258,46 @@
       ]
     }
   },
+  "clusterInference": {
+    "defaultClusterConfig": {
+      "clusterName": "default-cluster-prop",
+      "clusterUuid": "1234-5678-1234567",
+      "config": {
+        "gceClusterConfig": {
+          "zoneUri": "us-central1-a"
+        },
+        "masterConfig": {
+          "instanceNames": [
+            "test-node"
+          ],
+          "machineTypeUri": "n1-standard-16",
+          "numInstances": 1
+        },
+        "workerConfig": {
+          "instanceNames": [
+            "test-node"
+          ],
+          "machineTypeUri": "n1-standard-32",
+          "numInstances": 1
+        }
+      },
+      "status": {
+        "state": "STOPPED"
+      }
+    },
+    "defaultCpuInstances": {
+      "driver": "n1-standard-16",
+      "executor": [
+        {"name": "n1-standard-1", "vCPUs": 1},
+        {"name": "n1-standard-2", "vCPUs": 2},
+        {"name": "n1-standard-4", "vCPUs": 4},
+        {"name": "n1-standard-8", "vCPUs": 8},
+        {"name": "n1-standard-16", "vCPUs": 16},
+        {"name": "n1-standard-32", "vCPUs": 32},
+        {"name": "n1-standard-64", "vCPUs": 64}
+      ]
+    }
+  },
```

Comment:
For dataproc we have some lookup table for supported GPU instances.
You can find it under `supportedGpuInstances`.
Instead of listing each instance-type one by one, it automatically infer the number of CPU based on the name.
This was used when we pick a gPU machine for migration.
So, the point here is that it is going to be confusing that we have different ways to do almost the same thing.

Reply:
Thanks @amahussein. Because we are supporting only single series, we do not need the look up table as the instance type is always `n1-standard-{vCPUs}`.

Reply:
Updated the code to match existing flow.
Note: we cannot use the same JSON object because it denotes `supportedGpuInstances` instead of CPU instances.